contracts/staking/contracts/src/immutable/MixinHyperParameters.sol

```diff
@@ -0,0 +1,163 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/LibRichErrors.sol";
+import "@0x/contracts-utils/contracts/src/Ownable.sol";
+import "../interfaces/IStakingEvents.sol";
+import "../libs/LibStakingRichErrors.sol";
+import "./MixinConstants.sol";
+
+
+contract MixinHyperParameters is
+    IStakingEvents,
+    MixinConstants,
+    Ownable
+{
+    // Minimum seconds between epochs.
+    uint256 internal epochDurationInSeconds = 2 weeks;
```

Comment:
I think state upgrades will be more robust if we maintain all of the state variables in `MixinStorage`. Especially because we frequently have to reorder imports to avoid the linearization errors.